Removes several permanently closed breweries (Gordon Biersch, Rogue A…

jallend1 opened the pull request:
…lehouse, etc), updates Elysian TangleTown to reflect new ownership and name change, adds several breweries that have opened since this list was created

chrisjm commented:
Thanks for the contribution @jallend1! You didn't know this (I need to update the documentation, sorry about that), but we're trying to keep breweries in the dataset for historical purposes. In which case, I think we're going to keep the breweries in the dataset and mark the "brewery_type" as "closed". 

Could you make those changes and re-commit? Let me know if you have any questions. Thanks!

jallend1 commented:
Done! There were a couple of edge cases that I was a little murky on...

Elysian Tangletown is still in the same location, but no longer owned by Elysian and has changed its name to TangleTown Public House. I marked Elysian Tangletown as closed and created a new line for the TangleTown Public House.

'Chainline' closed moved locations, and also changed the name to 'Chainline Taproom at the Urban.' I marked Chainline closed, and added a new line for their current location. 

Wasn't quite sure where the line is on ownership / name / address changes deciding where marking a line as closed versus updating with its new information. Hopefully these were the correct approaches stylistically, but if not don't hesitate to send it back to me!

chrisjm commented:
Awesome! Thanks, @jallend1 ! Perfect!
